app/components/common/Navigation.ozar.vue

```diff
@@ -264,7 +264,9 @@
                     li(v-if="me.isAdmin()")
                       a.account-dropdown-item(href="/admin") {{ $t('account_settings.admin') }}
                     li(v-if="serverSession && serverSession.amActually")
```

Comment:
btw since you're using `spy` function to switch account, i think we have some places to use `serverSession && serverSession.amActually` to check spy by admin, now we should update them so that we won't make a normal user with related account to have admin permission. did you check every usage of serverSession?  -- this is blocking i think